### ISSUE TITLE
ds_weight is not changing during training

### ISSUE THREAD

**Issue by jecaJeca** [clovaai/stargan-v2-tensorflow#15]
Hi,

According to the official TF documentation, ds_weight decay implemented on this way will not affect the loss weight inside g_train function. Please, take a look here: https://www.tensorflow.org/guide/function#depending_on_python_global_and_free_variables
Have you observed that behavior in your experiments as well?

Maybe loss weights can be passed as an argument to this function or directly converted to tf.Variable ?
